examples/quiz-robust/qwiklabs.yml

```diff
@@ -0,0 +1,107 @@
+entity_type: Quiz
+schema_version:
+
+default_locale: en
+
+title: Robust Quiz
+passing_percentage: 67
+
+items:
+  - type: multiple-choice
+    stem:
+      locales:
+        en: Who won the Nobel Prize for discovering the cause of the photoelectric effect?
+        fr: Qui a remporté le prix Nobel pour avoir découvert la cause de l'effet photoélectrique?
+    options:
+      - title:
+          locales:
+            en: Albert Einstein
+            fr: Albert Einstein
+        is_answer: true
+        rationale:
+          locales:
+            en: Correct! Albert Einstein won the Nobel Prize in 1922 for work that forms the foundation for how modern solar panels work.
+            fr: Correct! Albert Einstein a reçu le prix Nobel en 1922 pour des travaux qui constituent le fondement du fonctionnement des panneaux solaires modernes.
```

Comment:
I don't think `Correct!` is french

Reply:
Actually, it is! It’s the masculine version of the word which according to Google Translate (and my shaky memory from HS french) is the correct (heh) gender in this context (ie where there’s no noun to modify).